components/coding/studentPortal/AssignmentInputBox.tsx

```diff
@@ -29,9 +29,9 @@ export const AssignmentInputBox: React.FC<AssignmentInputBoxProps> = ({
     (document.getElementById("input") as HTMLInputElement).value = "";
   }, []);
   return (
-    <div className="grid grid-cols-1 gap-4 p-6 bg-white shadow-lg dark:bg-gray-900 lg:grid-cols-resource-row">
+    <div className="grid grid-cols-1 gap-4 p-6 bg-white shadow-lg dark:bg-gray-900 ">
```

Comment:
wasn't sure if we should eventually do some kind of check to determine if this is a link or not and then do a pop-up if it isn't asking for a link.